content/faq/dev_use_current_revision_in_build.md

```diff
@@ -1,6 +1,8 @@
 ---
 description: Using environment variables in GoCD
 keywords: environment variable, GoCD configuration, GoCD tasks, custom command, shell-script, ruby script, 
+aliases:
+    - /faq/environment_variables.html
```

Comment:
@kritika-singh3 Please check how this works for versioned links. For example 19.2.0/faq/environment_variables.html. I made this change and had to revert the fix as it was breaking.